### PR TITLE
fix Meteor multiple servers .execute()

### PR DESCRIPTION
**Pull request by ejfrancis**
### Overview
* fixes so that the ddp `.execute()` method is added to each DDP instance
* fixes README example

### Explanation

Turns out I should try the example I add to the README before committing it next time 😞 

This example below that I had added didn't work because the `.execute()` method was only being added to the first DDP instance every time instead of each individual DPP instance. But also `process.env.PORT` isn't set to the front-facing port, it's the port of the app behind Meteor's local proxy. So I changed it to a test that asserts `process.env.ROOT_URL`. This new example actually works.

@samhatoum

### CODE REVIEW

File: src/lib/chimp-helper.js [hunk 1]
```diff
@@ -184,7 +184,7 @@ var chimpHelper = {
           }
         }, timeout);
         try {
-          result = server.call('xolvio/backdoor', func.toString(), args);
+          result = ddpInstance.call('xolvio/backdoor', func.toString(), args);
```
Comment (ejfrancis) [inline src/lib/chimp-helper.js]: add backdoor to each DDP instance

**Comment by ejfrancis** [comment]
not sure why it failed on Circle, it passes on local.  The error in Circle is strange and seems irrelevant to this PR's changes

```
Error: ENOENT: no such file or directory, open '/home/ubuntu/chimp/dist/__tests__/.babelrc'
    at Error (native)
```

**Comment by samhatoum** [comment]
rerun the tests. I'm trying to get to the bottom of that error :/

**Comment by ejfrancis** [comment]
do I have permission to in Circle? I can't figure out how to do it through the UI

**Pull request merged**